src/assets/translations/en-US.json

```
@@ -693,6 +693,33 @@
     "disputes.overview.ongoingAndClosed.errors.listEmpty": "No disputes found",
     "disputes.overview.ongoingAndClosed.errors.updateFilters": "Try different filters or check again later for new disputes.",
     "disputes.overview.ongoingAndClosed.limitSelect.a11y.label": "Disputes per page",
+    "payByLink.common.linkType.open": "Open",
+    "payByLink.common.linkType.singleUse": "Single Use",
+    "payByLink.common.status.ACTIVE": "Active",
+    "payByLink.common.status.COMPLETED": "Completed",
+    "payByLink.common.status.EXPIRED": "Expired",
+    "payByLink.common.status.PAYMENT_PENDING": "Payment pending",
+    "payByLink.details.title": "Pay By Link details",
+    "payByLink.overview.common.fields.amount": "Amount",
+    "payByLink.overview.common.fields.createdAt": "Creation Date",
+    "payByLink.overview.common.fields.currency": "Currency",
+    "payByLink.overview.common.fields.expirationDate": "Expiration",
+    "payByLink.overview.common.fields.id": "Payment Link ID",
+    "payByLink.overview.common.fields.linkType": "Type",
+    "payByLink.overview.common.fields.merchantReference": "Merchant Reference",
+    "payByLink.overview.common.fields.shopperEmail": "Shopper Email",
+    "payByLink.overview.common.fields.status": "Status",
+    "payByLink.overview.common.filters.types.statusGroup": "Status groups",
+    "payByLink.overview.errors.listEmpty": "No payment links found",
+    "payByLink.overview.errors.listUnavailable": "We couldn't load your payment link list.",
+    "payByLink.overview.filters.label": "Pay By Link filters",
+    "payByLink.overview.filters.types.amount.label": "Amount",
+    "payByLink.overview.filters.types.linkTypes.label": "Type",
+    "payByLink.overview.filters.types.merchantReference.label": "Merchant reference",
+    "payByLink.overview.filters.types.status.label": "Status",
+    "payByLink.overview.pagination.controls.limitSelect.label": "Payment links per page",
+    "payByLink.overview.pagination.label": "Pay by link table pagination",
+    "payByLink.overview.title": "Pay By Link Overview",
```

We use sentence case in other component titles.

```suggestion
    "payByLink.overview.title": "Pay By Link overview",
```